README.rst

```diff
@@ -101,6 +101,49 @@ migrated instance and a custom worker image name::
     "migr_image_name": "Nano"}'
 
 
+Source environment
+------------------
+
+A source environment is an optional parameter that defines
+provider-specific source parameters
+
+Network map
+-----------
+
+The network map is JSON mapping between identifiers of networks on the source
```

Comment:
Please replace this with:
`The network map is a JSON mapping between identifiers of networks on the source platform, each being associated with the identifier of a network on the destination. The values to be mapped depend on the source and destination provider plugins, respectively (ex: it may be the name of a network, or the full ID, etc...)`